vnext/Microsoft.ReactNative/Modules/PaperUIManagerModule.cpp

```diff
@@ -167,45 +167,47 @@ class UIManagerModule : public std::enable_shared_from_this<UIManagerModule>, pu
       React::JSValueArray &&point,
       std::function<void(double nativeViewTag, double left, double top, double width, double height)>
           &&callback) noexcept {
-    auto &node = m_nodeRegistry.getNode(reactTag);
-    float x = static_cast<float>(point[0]);
-    float y = static_cast<float>(point[1]);
-    m_nativeUIManager->findSubviewIn(node, x, y, std::move(callback));
+    if (auto node = m_nodeRegistry->findNode(reactTag)) {
+      float x = static_cast<float>(point[0]);
+      float y = static_cast<float>(point[1]);
+      m_nativeUIManager->findSubviewIn(*node, x, y, std::move(callback));
```

Comment:
I'm guessing the async caller on the JS side will still expect a result. E.g. calling the callback with null instead of no-op on not found.

Reply:
Would be worth checking if Android/iOS returns null vs calling an errback.

Reply:
https://github.com/facebook/react-native/blob/ff9c8cdac9b1734ad5f193116880b7b91e08bc6e/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIViewOperationQueue.java#L541

Looks like it calls the callback with no args.